connexion/mock.py

```diff
@@ -23,10 +24,15 @@ def resolve(self, operation):
         :type operation: connexion.operations.AbstractOperation
         """
         operation_id = self.resolve_operation_id(operation)
-        if not operation_id:
-            # just generate an unique operation ID
-            operation_id = f"mock-{self._operation_id_counter}"
-            self._operation_id_counter += 1
+        if operation_id is None:
+            operation_ref = operation.path + operation.method
+            if operation_ref in self._resolved_operations:
+                operation_id = self._resolved_operations.get(operation_ref)
```

Comment:
In this branch, we know it exists, so we should be able to directly use access by key?